generators/common/templates/src/main/asciidoc/db.adoc.ejs

```diff
@@ -0,0 +1,45 @@
+=== Database
+
+Production database is *<%_ if (databaseType === 'sql') { _%><%= prodDatabaseType %>%><%_ } else { _%><%= databaseType %><%_ } _%>*.
+
+.Databases configuration
+|===
+|Environment |URL |Login |Password
+
+| Local development
+<%_ if (databaseType === 'sql') { _%>
+<%_ if (devDatabaseType === 'mysql') { _%>
+| url: jdbc:mysql://localhost:3306/jhipster?useUnicode=true&characterEncoding=utf8&useSSL=false&useLegacyDatetimeCode=false&serverTimezone=UTC
```

Comment:
I wonder if we shouldn't add this to a variable in the subgenerator, and use the variable both here and in `application-dev.yml` to be sure the documentation doesn't diverge from the code.
Also this template could be moved to the `server` subgenerator.